src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -2205,8 +2222,19 @@ int xclLoadXclBin(xclDeviceHandle handle, const xclBin *buffer)
     LOAD_XCLBIN_CB ;
 #endif
     if (!ret) {
+      char Buff[1024] = {0};
```

Comment:
Dont hard code the size of char array. Take the char*. You are anyway getting size from memGroupInfo.size()

Reply:
ok. I will make the necessary changes.